jenkins/pipelines/ci/tikv/tikv_ghpr_test_nextest.groovy

```diff
@@ -0,0 +1,391 @@
+echo "release test: ${params.containsKey("release_test")}"
+if (params.containsKey("release_test")) {
+    echo "release test: ${params.containsKey("release_test")}"
+    ghprbTargetBranch = params.getOrDefault("release_test__ghpr_target_branch", params.release_test__release_branch)
+    ghprbCommentBody = params.getOrDefault("release_test__ghpr_comment_body", "")
+    ghprbActualCommit = params.getOrDefault("release_test__ghpr_actual_commit", params.release_test__tikv_commit)
+    ghprbPullId = params.getOrDefault("release_test__ghpr_pull_id", 0)
+    ghprbPullTitle = params.getOrDefault("release_test__ghpr_pull_title", "")
+    ghprbPullLink = params.getOrDefault("release_test__ghpr_pull_link", "")
+    ghprbPullDescription = params.getOrDefault("release_test__ghpr_pull_description", "")
+}
+
+def notRun = 1
+def chunk_count = 3
+
+def pod_image_param = ghprbTargetBranch
+
+
+// example hotfix branch  release-4.0-20210724 | example release-5.1-hotfix-tiflash-patch1
+// remove suffix "-20210724", only use "release-4.0"
+if (ghprbTargetBranch.startsWith("release-") && ghprbTargetBranch.split("-").size() >= 3 ) {
+    println "tikv hotfix branch: ${ghprbTargetBranch}"
+    def k = ghprbTargetBranch.indexOf("-", ghprbTargetBranch.indexOf("-") + 1)
+    ghprbTargetBranch = ghprbTargetBranch.substring(0, k)
+    pod_image_param = ghprbTargetBranch
+}
+
+if (!ghprbTargetBranch.startsWith("release-")) {
+    pod_image_param = "master"
+}
+println "ci image use  hub.pingcap.net/jenkins/tikv-cached-${pod_image_param}:latest"
+
+def run_test_with_pod(Closure body) {
+    def label = "${JOB_NAME}-${BUILD_NUMBER}"
+    def cloud = "kubernetes-ng"
+    def namespace = "jenkins-tikv"
+    def go_image = "hub.pingcap.net/jenkins/centos7_golang-1.13:cached"
+    podTemplate(label: label,
+            cloud: cloud,
+            namespace: namespace,
+            idleMinutes: 0,
+            containers: [
+                    containerTemplate(
+                        name: 'golang', image: "hub.pingcap.net/jenkins/tikv-cached-${pod_image_param}:latest",
```

Comment:
use `pod_image_param ` here, so the definition of this variable needs to be modified